problems/1.intro.js

```diff
@@ -4,6 +4,9 @@
  * @returns undefined
  */
 function helloWorld() {
+  console.log("hello world")
+  
+  return undefined
```

Comment:
No need to return undefined  explicitly, it will automatically be returned if there is not return.